src/app/auth/components/registration/registration.componet.ts

```diff
@@ -0,0 +1,181 @@
+import { Component, OnInit, computed, signal } from '@angular/core';
+import {
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+  FormsModule,
+  FormBuilder,
+} from '@angular/forms';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatSelectModule } from '@angular/material/select';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+
+import {
+  Router,
+  RouterLink,
+  RouterLinkActive,
+  RouterOutlet,
+} from '@angular/router';
+import { provideNativeDateAdapter } from '@angular/material/core';
+import { CustomValidatorsService } from '../../services/custom-validators.service';
+import { FormatDataService } from '../../../shared/services/format-date.service';
+import { SnackbarService } from '../../../shared/services/mat-snackbar.service';
+import {
+  AuthCustomerService,
+  CustomerRegestrationForm,
+} from '../../services/customer-auth.service';
+import { CustomerResponse } from '../../services/interfaces';
+import { AuthService } from '../../services/auth.service';
+
+@Component({
+  imports: [
+    MatFormFieldModule,
+    MatButtonModule,
+    MatInputModule,
+    MatIconModule,
+    MatSelectModule,
+    MatDatepickerModule,
+    MatSnackBarModule,
+    FormsModule,
+    ReactiveFormsModule,
+    RouterOutlet,
+    RouterLink,
+    RouterLinkActive,
+  ],
+  providers: [provideNativeDateAdapter()],
+  standalone: true,
+  selector: 'app-registration-page',
+  templateUrl: './registration.component.html',
+  styleUrls: ['./registration.component.scss'],
+})
+export class RegistrationComponent implements OnInit {
+  isAuthenticated = computed(() => {
+    return this.authService.isAuthenticated();
+  });
+
+  registrationForm: FormGroup;
+  submitInProcess = signal(false);
+  hidepassword = true;
+
+  countries = ['Poland', 'United States', 'Canada'];
+
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private fb: FormBuilder,
+    private CustomValidators: CustomValidatorsService,
+    private FormatData: FormatDataService,
+    private snackbarService: SnackbarService,
+    private authCustomerService: AuthCustomerService,
+  ) {
+    this.registrationForm = this.fb.group(
+      {
```

Comment:
It is better not to have anything in conscructor's {}. You can always use ngOnInit and initForm() there.